packages/wallet-core/src/bignumber.ts

```diff
@@ -49,7 +49,8 @@ export class BN {
   static toHexString = unaryOperator('toHexString');
 
   static from = (n: BigNumberish | BN): Uint256 => EthersBigNumber.from(n).toHexString() as Uint256;
-  static isBigNumber = val => typeof val === 'string' && !!val.match(/^0x[0-9A-Fa-f]{0,64}$/);
+  static isBigNumber = (val: any) =>
```

Comment:
Probably worth renaming this to `isUint256` -- maybe it's worth it to squeeze into this PR?